### PR TITLE
Adds troubleshooting instructions

### PR DESCRIPTION
**Pull request by JasonMFry**
#### Short description of what this resolves:

Adds troubleshooting instructions that could help many people get around common
error messages related to updating/uploading settings.

#### Changes proposed in this pull request:

- Updates Readme

**~~Fixes~~ Helps With**:
https://github.com/shanalikhan/code-settings-sync/issues/388, https://github.com/shanalikhan/code-settings-sync/issues/408, https://github.com/shanalikhan/code-settings-sync/issues/689, https://github.com/shanalikhan/code-settings-sync/issues/703

#### How Has This Been Tested?
It's just documentation, no testing required.

#### Screenshots (if appropriate):
N/A

#### Checklist:

- [x] I have read the [contribution](https://github.com/shanalikhan/code-settings-sync/blob/master/CONTRIBUTING.md#setup-extension-locally) guidelines.
- [x] My change requires a change to the documentation and Github Wiki.
- [x] I have updated the documentation and Wiki accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,15 @@ Here is the gif of the complete process when you execute the Upload command (Mig
 
 ![Upload](https://media.giphy.com/media/xT9IglKxSqs2Wdwq2c/source.gif)
 
+### Troubleshooting
+
+If for some reason you get an error when following these instructions try the following:
```
Comment (shanalikhan) [inline README.md]: public Wiki edit is enabled. What i suggest to make readme more extensive. 
We can give a link of public wiki in read me, and put all the possibilities there.

Comment (shanalikhan) [reply README.md]: You can create new page and put these things there and add link of troubleshooting here.

**Comment by shanalikhan** [comment]
I have created new page in Wiki 
https://github.com/shanalikhan/code-settings-sync/wiki/Troubleshooting

Here you can write the steps and send PR with the wiki link attached in troubleshooting section in readme

File: README.md (hunk 1 above)
Comment (JasonMFry) [reply README.md]: Done! https://github.com/shanalikhan/code-settings-sync/wiki/Troubleshooting#invalid-gist-id-or-invalid--expired-github-token

Feel free to close this PR if you'd like, or I will update it to point to the wikipage if you'd prefer.

**Pull request closed without merging**